OrchardHUN.TrainingDemo.Tests/StartHere.txt

```diff
@@ -13,7 +13,7 @@ Other gotchas:
 

```

Comment:
Will write it here because the mentioned file is not in the PR.
The PersonManagerTests.cs contains a URL: http://code.google.com/p/moq/
Moq is now in GitHub, so, you can replace this URL with the current one: https://github.com/Moq/moq4